Formula/mr2.rb

```diff
@@ -0,0 +1,18 @@
+class Mr2 < Formula
+  desc "Expose local server to external network"
+  homepage "https://github.com/txthinking/mr2"
+  url "https://github.com/txthinking/mr2/archive/refs/tags/v20210401.tar.gz"
+  sha256 "3cf2874a5945e79fd9ca270181de1a9d6a662434455c58e2e20e5dbfdebd64c7"
+  license "GPL-3.0-only"
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", *std_go_args(ldflags: "-s -w"), "./cli/mr2"
+  end
+
+  test do
+    version = shell_output("#{bin}/mr2 -v")
+    assert_match version.to_s, version, "Unexpected output of version"
```

Comment:
Can you propose a more substantial test than checking the version or calling for help? These tests rarely ever fail even for broken installations.

>We want tests that don’t require any user input and test the basic functionality of the application. For example `foo build-foo input.foo` is a good test and (despite their widespread use) `foo --version` and `foo --help` are bad tests. 

Ref. https://docs.brew.sh/Formula-Cookbook#add-a-test-to-the-formula